cf/field.py

```diff
@@ -626,15 +626,15 @@ def analyse_items(self, relaxed_identities=None):
      'aux_coords': {'N-d': {'aux0': <CF AuxiliaryCoordinate: latitude(110, 106) degrees_N>,
                             'aux1': <CF AuxiliaryCoordinate: longitude(110, 106) degrees_E>},
                     'dim0': {'1-d': {},
-                             'N-d': {}},
+                             'N-d': {},},
```

Comment:
Would you recommend we now do this, or similar, to prevent natural ``}}`` cases in docstrings, then, say if writing a new one? I only see the test to detect unprocessed `{{ ... }}` expressions fail with both opening and closing double braces , but I suppose this is to be safe in case somehow the former could get added beforehand?

Reply:
@sadielbartholomew I think this is now not necessary - as we have dropped the test for `}}` in the docstrings. It seemed unfair to test for it, as it is valid python (unlike `{{`)